common/armies/hfe_armies.txt

```diff
@@ -38,3 +38,310 @@
 # 		}
 # 	}
 # }
+
+hfe_army_high_mechanicus_bodyguards = {
+	defensive = yes
+	is_pop_spawned = no
+	pop_limited = no
+	health = 1.1
+	damage = 2
+	morale = 5
+	has_morale = yes
+	morale_damage = 1.5
+	collateral_damage = 0.9
+	war_exhaustion = 2
+	icon_frame = 5
+	has_species = yes
+
+	resources = {
+		category = armies
+		cost = {
+			trigger = {
+				owner = {
+					NOT = {
+						is_primitive = yes
+						is_country_type = primitive
+					}
+				}
+			}
+			energy = 500
+			food = 30
+			alloys = 300
+		}
+		cost = {
+			alloys = 100
+		}
+		produces = {
+			unity = 1
+		}
+		upkeep = {
+			food = 2
+			alloys = 2
+			energy = 2
+			minerals = 2
+		}
+	}
+
+	potential = {
+		owner = { has_country_flag = hfe_varelse_primitive_0 }
+		owner = {
+			OR = {
+				has_civic = hfe_civic_high_mechanicus
+				has_country_flag = high_mechanicus_retained
+				is_spiritualist = yes
+			}
+			NOT = {
+				has_civic = hfe_civic_overthrown_mechanicus
+				has_country_flag = varelse_fedora_governance
```

Comment:
It says "Varelse", which is unique to the Mod.

I apologize for that omission and I should have included it. However, it's not like I can go through dozens of files and change it back every time I see it. That particular flag is EVERYWHERE and very important to my changes. That would break a lot of mechanics and issues. I occasionally forgot to add the prefix and for that, I am sincerely sorry.

Reply:
You can use auto replace

Reply:
Okay, done